web-ui/src/components/routes/NewUserRoutes.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from "react";
```

Comment:
Delete the unused 'useContext' so the gradle build does not fail.

`import React from "react";`